templates/gcp/index.html

```diff
@@ -0,0 +1,189 @@
+        {{
+          image(
+            url="https://assets.ubuntu.com/v1/c2307ee1-Google+Cloud.png",
+            alt="",
+            width="375",
+            height="240",
+            hi_def=True,
+            loading="auto",
+          ) | safe
+        }}
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-12">
+      <h2 class="p-muted-heading u-align--center">UBUNTU ON GCP POWERS CLOUD LEADERS LIKE</h2>
+      <ul class="p-inline-images u-no-margin--bottom">
+        <li class="p-inline-images__item">
+          {{
+            image(
+              url="https://assets.ubuntu.com/v1/b3315d88-walmart.svg",
+              alt="Walmart",
+              height=60,
+              width=200,
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-inline-images__logo"},
+            ) | safe
+          }}
+        </li>
+        <li class="p-inline-images__item">
+          {{
+            image(
+              url="https://assets.ubuntu.com/v1/2400d5a7-Spotify_logo_with_text.svg",
+              alt="Spotify",
+              height=60,
+              width=200,
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-inline-images__logo"},
+            ) | safe
+          }}
+        </li>
+        <li class="p-inline-images__item">
+          {{
+            image(
+              url="https://assets.ubuntu.com/v1/1d3f5d21-Netflix_2015_logo.svg",
+              alt="Netflix",
+              height=43,
+              width=200,
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-inline-images__logo"},
+            ) | safe
+          }}
+        </li>
+      </ul>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-6">
+      <h2>Choose the right Ubuntu for you</h2>
+      <p>Select between the standard Ubuntu Server and the Pro edition, which includes expanded security, compliance and GCP integration:</p>
+    </div>
+  </div>
+
+  <div class="row u-equal-height">
+    <div class="col-6 p-card">
+      <h3 class="p-card__title">Ubuntu</h3>
+      <p class="p-heading--two">Free to use</p>
+      <hr class="u-sv1">
+      <div class="p-card__content">
+        <ul class="p-list">
+          <li class="p-list__item is-ticked">GCP-optimised kernel and userspace components built from the latest releases</li>
+          <li class="p-list__item is-ticked">Broad set of open source application components available for development</li>
+          <li class="p-list__item is-ticked">Security patches for the kernel and key infrastructure components</li>
+          <li class="p-list__item is-ticked">Updates mirrored and images published in all GCP regions worldwide</li>
+          <li class="p-list__item is-ticked">5-year lifetime</li>
+        </ul>
+        <p>Run Ubuntu 18.04 LTS: Standard (Coming soon)</p>
```

Comment:
```suggestion
        <p>Run Ubuntu 18.04 LTS: Standard &nbsp;<span class="p-label--updated">Coming soon</span></p>
```

Reply:
This has been replaced by a link to the marketplace